java/src/com/ibm/streamsx/topology/context/remote/RemoteContext.java

```diff
@@ -16,7 +16,8 @@
     public enum Type {
         TOOLKIT,        
         BUILD_ARCHIVE,
-        REMOTE_BUILD_AND_SUBMIT
+	REMOTE_BUILD_AND_SUBMIT,
```

Comment:
I think we want to remove this context and instead have a parameter in the config that specifies to force use of a remote build. This is because this will become an option on multiple contexts, e.g. if I just want to build a bundle using the BUNDLE context.